non-software.md

```diff
@@ -18,6 +18,14 @@ Any open source software license or open license for media (see [above](#data-me
 
 The [SIL Open Font License 1.1](/licenses/ofl-1.1/) keeps fonts open, allowing them to be freely used in other works.
 
+### Hardware
+
+There are the CERN Open Hardware family of licenses: [CERN-OHL-P-2.0](/licenses/cern-ohl-p-2.0/), the permissive variant ([MIT](/licenses/mit/)-like); [CERN-OHL-W-2.0](/licenses/cern-ohl-w-2.0/), the “weakly reciprocal” variant ([MPL](/licenses/mpl-2.0/)-like), and [CERN-OHL-S-2.0](/licenses/cern-ohl-s-2.0/), the “strongly reciprocal” variant ([GPL](/licenses/gpl-3.0/)-like).  The CERN OHL family is also applicable to HDL source code and synthesized bitstreams as would be used in FPGAs.
```

Comment:
```suggestion
The CERN Open Hardware family of licenses includes [CERN-OHL-P-2.0](/licenses/cern-ohl-p-2.0/), the permissive variant; [CERN-OHL-W-2.0](/licenses/cern-ohl-w-2.0/), the “weakly reciprocal” variant, and [CERN-OHL-S-2.0](/licenses/cern-ohl-s-2.0/), the “strongly reciprocal” variant..
```

I'd skip the inexact software comparisons. The last sentence seems weirdly specific, is there a reason one would assume the licenses are not applicable to HDL relative to other hardware designs?

Reply:
Fine with me. One of the main reasons for coming up with version 2 of CERN OHL was the lack of good/usable reciprocal licenses for HDL code. Both GPL and LGPL have serious problems when applied to HDL code as used e.g. for FPGA and ASIC design. HDL is kind of half-way between software and hardware, and as such it needs a bit of guidance when it comes to licensing. Again, I don't know if this educational bit belongs here.

Reply:
I guess https://ohwr.org/project/cernohl/wikis/FAQ#questions-about-fpga-and-asic-design covers this? I would not mind linking to that (or to the FAQ in general), but I wonder if "synthesized bitstreams" should be included? That term isn't used in the FAQ and doesn't seem to be a very common term based on web search results.

Reply:
I agree. Linking to the FAQ chapter on FPGA and ASIC design would provide all the interesting information in this domain. I also agree about "synthesized bitstreams". Bitstreams are the end result of a process which includes more than just synthesis. I wrote a bit about it here: https://www.oshwa.org/best-practices-for-sharing-fpga-designs/
A quick fix would be to replace "synthesized bitstreams" by "bitstreams".